pweave/processors/base.py

```diff
@@ -125,7 +128,7 @@ def _runcode(self, chunk):
 
 
         if chunk['type'] == 'code':
-            sys.stdout.write("Processing chunk %(number)s named %(name)s from line %(start_line)s\n" % chunk)
+            logger.warn("Processing chunk %(number)s named %(name)s from line %(start_line)s" % chunk)
```

Comment:
It definitely should be not a warning. Processing of a code chunk is exactly what we expect that code to do.

Reply:
info, agreed